.github/release.yml

```diff
@@ -0,0 +1,20 @@
+changelog:
```

Comment:
do we want this functionality of releasing in a semver manner? you need to add a label if minor major etc release like on hyku
if not, delete this file

Reply:
going to keep this file to keep this as close as possible to hyku